ui/desktop/tests/acceptance/projects/settings/index-test.js

```diff
@@ -9,10 +9,12 @@ import { setupApplicationTest } from 'ember-qunit';
 import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 import { authenticateSession } from 'ember-simple-auth/test-support';
 import WindowMockIPC from '../../../helpers/window-mock-ipc';
+import setupStubs from 'api/test-support/handlers/cache-daemon-search';
 
 module('Acceptance | projects | settings | index', function (hooks) {
```

Comment:
Should we have tests for the color theme picker here now instead? Maybe we should have the moving of the theme picker in a different PR completely

Reply:
moved it outside this PR as the tests might need some refactors to follow our most recent patterns